keepercommander/commands/enterprise.py

```diff
@@ -3037,6 +3047,122 @@ def get_filter(filter_value, convert):
+                                if user['key_type'] == 'no_key':
+                                    username = user['encrypted_data']
+                            username = email if (username is None or not username.strip()) else username
+                            self.user_lookup[user['enterprise_user_id']] = { 'username': username, 'email': email }
+
+        info = {
+            'username': enterprise_user_id,
+            'email': enterprise_user_id
+        }
+
+        if enterprise_user_id in self.user_lookup:
+            info = self.user_lookup[enterprise_user_id]
+
+        return info
+
+    def execute(self, params, **kwargs):
+        if kwargs.get('syntax_help'):
+            logging.info(security_audit_report_description)
+            return
+
+        format = 'table'
+        if kwargs.get('format'):
+            format = kwargs['format']
+
+        rq = proto.SecurityReportRequest()
+        rs = api.communicate_rest(params, rq, 'enterprise/get_security_report_data')
+
+        security_report_data_rs = proto.SecurityReportResponse()
+        security_report_data_rs.ParseFromString(rs)
+
+        rows = []
+        for sr in security_report_data_rs.securityReport:
+            user_info = self.resolve_user_info(params, sr.enterpriseUserId)
+            user = user_info['username'] if 'username' in user_info else str(sr.enterpriseUserId)
+            email = user_info['email'] if 'email' in user_info else str(sr.enterpriseUserId)
+            twofa_on = False if sr.twoFactor == 'two_factor_disabled' else True
+            row = {
+                'username': user,
+                'email': email,
+                'weak': 0,
+                'medium': 0,
+                'strong': 0,
+                'reused': sr.numberOfReusedPassword,
+                'unique': 0,
+                'securityScore': 25,
+                'twoFactorChannel': 'Off' if sr.twoFactor == 'two_factor_disabled' else 'On'
+            }
+            master_password_strength = 1
+
+            if sr.encryptedReportData:
+                sri = rest_api.decrypt_aes(sr.encryptedReportData, params.enterprise['unencrypted_tree_key'])
+                data = json.loads(sri)
+                row['weak'] = data['weak_record_passwords'] if 'weak_record_passwords' in data else 0
+                row['strong'] = data['strong_record_passwords'] if 'weak_record_passwords' in data else 0
+                row['medium'] = data['total_record_passwords'] - row['weak'] - row['strong']
+                row['unique'] = data['total_record_passwords'] - row['reused']
+                score = self.get_security_score(data['total_record_passwords'], row['strong'], row['unique'], twofa_on, master_password_strength)
+                score = int(100 * round(score, 2))
+                row['securityScore'] = score
+            rows.append(row)
+
+        fields = ['username', 'email', 'weak', 'medium', 'strong', 'reused', 'unique', 'securityScore', 'twoFactorChannel']
```

Comment:
it would be great to have human readable columns in the table output, for example `securityScore` -> `Security Score`